extension/pybindings/portable_lib.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+
+# When installed as a pip wheel, we must import `torch` before trying to import
+# the pybindings shared library extension. This will load libtorch.so and
+# related libs, ensuring that the pybindings lib can resolve those runtime
+# dependencies.
+import torch as _torch
```

Comment:
Do we rely on ATen for the pybind? I'm guessing including torch here is going to pull all of that in. Okay, but just checking

Reply:
We do, so that users can use normal torch tensors for their inputs and outputs, making it easier for them to use.